Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:14.10.0-alpine
+
+ENV NODE_VERSION 14.10.0
```

Comment:
This env variable is unused?
If so, I guess this can go.
Will make maintenance easier, this the line above will be updated automatically by renovate.